lib/home/profile/account/account_settings.dart

```diff
@@ -65,10 +95,46 @@ class _AccountSettingsState extends State<AccountSettings> {
                   ),
                 ),
               ),
-            _ => const ListTile(
-                leading: Icon(Icons.person_off_outlined),
-                title: Text('close account'),
-                // onTap: () {},
+            _ => ListTile(
+                leading: const Icon(Icons.person_off_outlined),
+                title: const Text('close account'),
+                onTap: () => showDialog(
+                  context: context,
+                  builder: (_) => AlertDialog(
+                    title: const Text('Close Account'),
+                    content: Column(
+                      mainAxisSize: MainAxisSize.min,
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        const Text(
+                          'To confirm deletion, please type "DELETE" in the field below and tap "Confirm".',
+                          style: TextStyle(fontSize: 16),
+                        ),
+                        TextField(
+                          controller: _deleteController,
+                          decoration: const InputDecoration(
+                            hintText: 'Type DELETE here',
+                            labelText: 'Confirmation',
+                          ),
+                        ),
```

Comment:
I love this idea!

In fact, I've been talking with Abigail about implementing a dialog like this one in multiple places throughout the repo.